addon/ng2/blueprints/ng2/files/karma.conf.js

```diff
@@ -46,5 +52,11 @@ module.exports = function(config) {
     autoWatch: true,
     browsers: ['Chrome', 'Firefox'],
     singleRun: false
-  });
+  };
+
+  if(process.env.TRAVIS){
+    configuration.browsers = ['Chrome_travis_ci', 'Firefox'];
+  }
+
```

Comment:
I'd prefer to keep this logic in the CLI's tests, the if condition is already there to force Firefox